### PR TITLE
update "requires at least"  value to 5.6

### PR DESCRIPTION
**Pull request by bph**
of plugin file.

## Description
For plugin, the minimum requirement is 5.5 - it's already updated in the readme.txt but not in the plugin header of gutenberg.php

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Review by ntsekouras — COMMENTED** [review]
It's 5.6 now :) - https://github.com/WordPress/gutenberg/pull/29701

**Comment by gziolo** [comment]
I think it should be 5.6 now after @youknowriad merged https://github.com/WordPress/gutenberg/pull/29701 😄

File: gutenberg.php
Comment (youknowriad) [inline gutenberg.php:6]: Do you think we can remove this entirely from here? I mean we already have this value defined in readme.txt isn't that enough?

Comment (swissspidy) [reply gutenberg.php:6]: I think we should keep them. See https://core.trac.wordpress.org/ticket/46938

Comment (swissspidy) [reply gutenberg.php:6]: Could add `Tested up to` here too, fwiw

**Comment by bph** [comment]
Wasn't fast enough 🤣😂 @youknowriad took care of it. 👍🦄

**Comment by bph** [comment]
Oh, sorry. Yes, I can update to 5.6. Hang-on

**Comment by youknowriad** [comment]
@bph can you also add the two headers from the readme.txt to avoid duplication?

**Comment by bph** [comment]
Updated @youknowriad -

File: gutenberg.php
Comment (gziolo) [reply gutenberg.php]: Thanks for clarifying. I will make sure we update `@wordpress/create-block` and its templates to follow the same approach.

**Comment by youknowriad** [comment]
Looks like this PR needs a rebase now (conflict with the readme update on trunk). So you can do something like `git fetch origin && git rebase origin trunk` and follow the instructions.

**Review by bph — COMMENTED** [review]
Should be fixed now  @youknowriad  --  it still shows a conflict?

**Comment by youknowriad** [comment]
@bph I can take care of it.

**Review by youknowriad — APPROVED** [review]

**Comment by youknowriad** [comment]
Looks we have some issues with wp-env at the moment, I'll merge this after we solve these.

**Pull request merged**
